<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/table.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="score">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>521</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI';&quot;&gt;50&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ваш баланс:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BlackJack</string>
   </property>
  </widget>
  <widget class="QLabel" name="p_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="p_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/eight_kr.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="p_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_kr.JPG</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="points_p">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="d_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="d_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="d_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QTextBrowser" name="points_d">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Взяти 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="p_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="d_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/dama_bub.JPG</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Завершити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Взяти 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>525</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Передати хід</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>293</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>293</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Почати гру</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>261</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть ставку та почніть гру</string>
   </property>
  </widget>
  <widget class="QLabel" name="p_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Picture/back.JPG</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>SecondWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
